circle.yml

```diff
@@ -26,6 +26,7 @@ test:
       -v $CIRCLE_TEST_REPORTS:/test_artifacts
       -v ~/cache/GeoLite2-Country.mmdb:/app/GeoLite2-Country.mmdb
       normandy:build py.test --junitxml=/test_artifacts/pytest.xml
+    - flake8 normandy
```

Comment:
You'll need to run this inside the docker container, if you want it to have any python requirements.